test/pylib/manager_client.py

```diff
@@ -239,3 +242,31 @@ async def get_host_id(self, server_id: ServerNum) -> HostID:
         except Exception as exc:
             raise Exception(f"Failed to get local host id address for server {server_id}") from exc
         return HostID(host_id)
+
+    async def server_sees_others(self, server_id: ServerNum, count: int, deadline: int = 45):
```

Comment:
The meaning of `deadline` in `wait_for` is a time point (and note it's a float), not a time interval, we should consistently stick to one meaning if we're using the same word.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/1b146e7296dab3a3d9df866582c92340727b7070